arrow-core/src/main/kotlin/arrow/core/extensions/const/divide/ConstDivideInstance.kt

```diff
@@ -54,17 +50,22 @@ fun <O, A, B, Z> divide(
 @Deprecated(
   "@extension kinded projected functions are deprecated",
   ReplaceWith(
-    "product(MO, arg1)",
-    "arrow.core.product"
+    "mapN(MO, arg0, arg1, arg2).map(arg3)",
+    "arrow.core.mapN"
```

Comment:
```suggestion
    "Const.mapN(MO, arg0, arg1, arg2).map(arg3)",
    "arrow.core.Const"
```

Idem for others in files.